### PR TITLE
make device infos settable

### PR DESCRIPTION
**Pull request by FabianTerhorst**

### CODE REVIEW

**Comment by AbandonedCart** [comment]
A method to set device info is necessary, but this is a nightmare to implement.

```
DeviceInfos myInfo = new DeviceInfos() {
    @Override
    String getAndroidBoardName() {
        return myAndroidBoardName;
    }
    @Override
    String getAndroidBootloader() {
        return myAndroidBootloader;
    }

…
    @Override
    String getFirmwareType() {
        return myFirmwareType;
    }
    @Override
    String getFirmwareFingerprint() {
        return myFirmwareFingerprint;
    }
};
```

**Comment by FabianTerhorst** [comment]
We need an interface for that and we don´t want duplicate ways for implementations.

**Comment by FabianTerhorst** [comment]
We decided that, because we want to provide an interface for the platform.

**Comment by FabianTerhorst** [comment]
updated the pr.

**Comment by FabianTerhorst** [comment]
now its possible to use 

``` java
DeviceInfo deviceInfo = new DeviceInfo(); 
deviceInfo.set...("");
go.setDeviceInfo(deviceInfo)
```

File: library/src/main/java/com/pokegoapi/api/PokemonGo.java [hunk 1]
```diff
@@ -192,4 +198,11 @@ public Map getMap() {
 		}
 		return map;
 	}
+
+	public SignatureOuterClass.Signature.DeviceInfo getDeviceInfo() {
```
Comment (jabbink) [inline library/src/main/java/com/pokegoapi/api/PokemonGo.java:207]: JavaDoc

**Pull request merged**